CHANGELOG.md

```diff
@@ -121,28 +255,21 @@
 ## Contributors
 
 * Andrew Block (@sabre1041)
-* Asra Ali (@asraa)
```

Comment:
Did the changelog accidentally updated old contributors?

Reply:
oopsy thanks for the catch

Reply:
thanks updated